nodejs-week1/app.js

```diff
@@ -0,0 +1,25 @@
+import express from 'express';
```

Comment:
you need to split your PR into two, one for week 1 and one for week 2 if you are having any troubles please ping me on slack to explain how.